docs/races.md

```diff
@@ -0,0 +1,204 @@
+# Races
+
+Gets list of races 
+
+**URL** : `/ergast/f1/races`
+
+---
+
+## Route Parameters:
+
+### Season
+
+**Filters for races only from a specified season. Year numbers are valid as is 'current' to get the current season**
+
+`/{season}/` -> ex: `/ergast/f1/2024/races`
+
+**Note**: To utilize the `season` parameter it needs to be the first route after `/ergast/f1/`
+
+---
+
+### Circuit
+
+**Filters for only races featuring a specified circuit**
+
+`/circuits/{circuitId}/` -> ex: `/ergast/f1/circuits/monza/races`
+
+---
+
+### Constructor
+
+**Filters for only races featuring a specified constructor**
+
+`/constructors/{constructorId}/` -> ex: `/ergast/f1/constructors/williams/races`
+
+---
+
+### Driver
+
+**Filters for only races featuring a specified driver**
+
+`/drivers/{driverId}/` -> ex: `/ergast/f1/drivers/hamilton/races`
+
+
+---
+
+### Grid
+
+**Filters for only races featuring a specified grid position**
+
+`/grid/{gridPosition}/` -> ex: `/ergast/f1/grid/27/races`
+
+---
+
+### Status
+
+**Filters for only races featuring a specified finishing status of a driver**
+
+`/status/{statusId}/` -> ex: `/ergast/f1/status/2/races`
+
+---
+
+## Query Parameters:
+
+`limit` - Allows you to alter the limit the number of results returned. Defaults to 30.
+
+`offset` - Allows you to offset the results by the specified number. Defaults to 0.
+
+---
+
+## Success Response
+
+**Code** : `200 OK`
+
+**Response Fields** :
+
+`MRData` : The root object of the json response.
+
+`MRData.series` : The racing series that the races are a part of.
+
+`MRData.total` : The total number of races available.
```

Comment:
Could something similar be done here also? series, total, limit, offset, url, etc all mean the same thing so moving it to a common file which we can link to would be great.